packages/snap/src/wallet/handlers/WalletSuggestChain.ts

```diff
@@ -0,0 +1,38 @@
+import { panel } from '@metamask/snaps-sdk';
+import type { JsonRpcRequest } from '@metamask/utils';
+import type {
+  RPCMethods,
+  SnapRequestParams,
+  SnapRequestsResponses,
+} from '@rarimo/rarime-connector';
+
+import { addChain, validateChain } from '@/wallet/chain';
+import { getChainPanel } from '@/wallet/ui';
+
+export const walletSuggestChain = async ({
+  request,
+  origin,
+}: {
+  request: JsonRpcRequest;
+  origin: string;
+}): Promise<SnapRequestsResponses[RPCMethods.WalletSuggestChain]> => {
+  const { chainInfo } =
```

Comment:
mb gettin rarimo chain from snap base instance will not be apropriate for using with rarimo wallet